studio/schemas/documents/backgroundImgwithCard.js

```diff
@@ -18,5 +18,54 @@ export default {
             title: 'Background Image',
             type: 'image',
         },
+        {
+            name: 'cardType',
+            title: 'Card Type',
+            type: 'string',
+            description:
+                'Selection will give card and/or button a primary or secondary color scheme.',
+            options: {
+                list: [
+                    { title: 'Primary', value: 'primary' },
+                    { title: 'Secondary', value: 'secondary' },
+                ],
+                layout: 'radio',
+            },
+        },
+        {
+            name: 'backgroundColor',
+            title: 'Background Color',
+            type: 'string',
+            description:
+                'Selection will give card and/or button a white, blue or transparent background color.',
+            options: {
+                list: [
+                    { title: 'Light', value: 'light' },
+                    { title: 'Dark', value: 'dark' },
+                    { title: 'Transparent', value: 'transparent' },
+                ],
+            },
+        },
+
+        {
+            name: 'cardOrientation',
+            title: 'Card Orientation',
+            type: 'string',
+            description: 'Selection will align the card on the right.',
+            options: {
+                list: [{ title: 'Right', value: 'right' }],
+                layout: 'radio',
+            },
+        },
+        {
+            name: 'button',
+            title: 'Button',
+            type: 'string',
+            description: 'Selection will give card a button',
+            options: {
+                list: [{ title: 'hasButton', value: 'hasButton' }],
+                layout: 'radio',
+            },
+        },
```

Comment:
Will these always be internal links? If so we'll need a way to pass a string for the url. We can probably forgo the `button` field and add one for the url. Should be able to evaluate that field (an empty string evaluates to `false`) to control button/cta display logic.